### ISSUE TITLE
Last 24 hours not today

### ISSUE THREAD

**Issue by derivadow** [nightscout/nightguard#102]
Because data is graphed from midnight to midnight (for the current day) it means you have no historical data soon after midnight + reduced data for the entire day. It would be better if the data was displayed as the previous 24 hours.

**Comment by dhermanns** [comment]
Hi Tom!

You will have 24 hours of historical data right after midnight: that is the
gray line. You will have to swipe to the right to see it.

Tom Scott <notifications@github.com> schrieb am Sa. 8. Aug. 2020 um 01:20:

**Comment by derivadow** [comment]
Hi, that's fair enough. Nonetheless the ux is quite different before and after midnight: at just after midnight you get 24 hours of data, just before 48 hours of data (and so easy to cross reference). IMHO a 24 hour view would be an improvement.

**Comment by dhermanns** [comment]
Yes - that‘s right. But therefore a major redesign. And right now I have
other things on the list that can‘t be done at all. But maybe someone else
would like to implement it. I will leave this open as a feature request.

**Comment by dhermanns** [comment]
Thinking about this more: What I love about the current implementation is: You could swipe to the left and will always stop at the beginning of the day. That would change when having the sliding 24h window. And that what is always confusing for me when using the nightscout website.

**Comment by derivadow** [comment]
I can see 100% this isn't a priority (only affects people with poor sleep patterns and those on night shifts). And yes I agree about the advantage vs the Nightscout site. I wonder if that is why other systems provide both views (last 24 hours + per day).
